src/Project/Classes/Stores/Orders/OnlineOrder.java

```diff
@@ -0,0 +1,52 @@
+package Project.Classes.Stores.Orders;
+
+import Project.Classes.Stores.Adress;
+import Project.Classes.Stores.Products.Product;
+import Project.Classes.Stores.Store;
+import Project.Classes.Users.Costumer;
+
+import java.util.ArrayList;
+
+public class OnlineOrder extends Order {
+    private Adress delivaryAdress;

```

Comment:
[Fix] o comanda care nu este data online, nu are adresa de livrare?

Reply:
Pai ma gandeam ca daca nu e data online, atunci este inregistrata comanda ca facuta in magazin

Reply:
ah, tu zici ca a comandat si a ridicat personal comanda? ok

Reply:
Daaa